### DISCUSSION TITLE
Run with --listen --api

### DISCUSSION THREAD

**Discussion by robumb** [AbdBarho/stable-diffusion-webui-docker#689] [Q&A]
I am a noob when it comes to docker. i have just installed the stable-diffusion-webui-docker on my windows machine and i want to integrate it with open webui. I've seen in another tutorial (installation without docker) that you have to start with `./webui.sh --list --api` in order to integrate with open webui.
Can anyone explain how i can add those tags for my installation?

**Comment by DevilaN** [comment]
That are parameters added when starting script. Here you can edit them in `docker-compose.yml` file in `environment:` section of proper service.
